src/lib/DeFiScriptErrors.sol

```diff
@@ -10,5 +10,5 @@ library DeFiScriptErrors {
     error InvalidInput();
     error TransferFailed(bytes data);
     error ApproveAndSwapFailed(bytes data);
-    error TooMuchSlippage();
+    error TooMuchSlippage(uint256 actualBuyAmount, uint256 expectedBuyAmount);
```

Comment:
nit: we should flip the order to be consistent with `TransactionTooExpensive(uint256 expectedMaxCost, uint256 actualMaxCost)`, which has expected first, then actual

Reply:
doh, yes